pydbtools/utils.py

```diff
@@ -59,6 +59,10 @@
         r"^[0-9]+$",  # numeric
         None,
     ),
+    (
+        r"[A-Za-z0-9]+:GitHubActions",  # GitHub action for e2e
```

Comment:
i put lower case letters in to stop this happening again, but not sure if they are ever needed